resources/assets/uad_lists.json

```
@@ -36909,5 +36909,125 @@
     "neededBy": [],
     "labels": [],
     "removal": "Advanced"
+  },
+  "com.android.connectivity.resources.overlay": {
+    "list": "Oem",
+    "description": "Cause bootloop?\nFound in TECNO Phone in overlay there's notify No Internet As Dialog When High Priority.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Expert"
```

```suggestion
    "removal": "Unsafe"
```
If it might cause a bootloop, then maybe `Unsafe` is a better option?

---

I wanna to someone test it, if its only on tecno phone.

---

Better not take risks, right?

---

yes so now in unsafe it will be

---

Yeah.